app/viewmodels/checkAnswers/DistributionSummary.scala

```diff
@@ -36,7 +36,7 @@ object DistributionSummary {
           .get(DistributionsIncludedPage)
           .map(form =>
             if (form.distributionsIncludedAmount.exists(_ > 0)) {
-              s"£${NumberFormat.getNumberInstance(Locale.UK).format(form.distributionsIncludedAmount.getOrElse(0))}"
+              s"£${NumberFormat.getNumberInstance(Locale.UK).format(form.distributionsIncludedAmount.get)}"
             } else {
               messages("distributionsIncluded.emptyValue")
             }
```

Comment:
Could be rewritten to avoid get

```
form.distributionsIncludedAmount.filter(_ > 0).map(a => s"£${NumberFormat.getNumberInstance(Locale.UK).format(a)}").getOrElse(messages("distributionsIncluded.emptyValue"))
```